docs/vm-ee-guide.md

```diff
@@ -0,0 +1,324 @@
+# Onboard
+
+## Table of Contents
+
+<!-- toc -->
+- [VirtualMachines](#virtualmachines)
+    - [Labels](#labels)
+    - [Cloud Tags](#cloud-tags)
+    - [Sample VirtualMachine Output](#sample-virtualmachine-output)
+        - [AWS](#aws)
+        - [Azure](#azure)
+- [ExternalEntity](#externalentity)
+    - [Sample ExternalEntity Output](#sample-externalentity-output)
+        - [AWS](#aws-1)
+        - [Azure](#azure-1)
+    - [Labels](#labels-1)
+    - [Cloud Tags](#cloud-tags-1)
+        - [Usage](#usage)
+<!-- /toc -->
+
+## Virtual Machine
+
+Virtual Machines are onboarded by specifying the matching criteria in `CloudEntitySelector` CR. The virtual machines from
+cloud are imported as `VirtualMachine` objects in `CloudEntitySelector` namespace. `VirtualMachine` object `Name` is
+unique resource name within the kubernetes cluster. In Azure, ascii sum of the characters in VM Resource ID added as
+suffix to name of the Virtual Machine and in AWS, instance ID of the Virtual Machine is used as `Name`.
```

Comment:
object `Name` is a unique resource
Kubernetes cluster
In Azure, the ASCII sum of all characters in the VM Resource ID is added as a suffix to the Virtual Machine name. In AWS, the instance ID of the Virtual Machine is used as the `Name`."

Reply:
done